src/test/java/com/streisky/virtualshowcasebackend/constants/AccountTestsConstants.java

```diff
@@ -0,0 +1,16 @@
+package com.streisky.virtualshowcasebackend.constants;
+
+import com.streisky.virtualshowcasebackend.domain.account.dto.AccountDTO;
+import com.streisky.virtualshowcasebackend.domain.account.entity.Account;
+
+public class AccountTestsConstants {
```

Comment:
A suggestion on the naming would be AccountMocks

Reply:
I don't know if this make sense, because the idea is just centralize constants used in the account test.

In this moment has only the controller, but in the future I'll do the CRUD also.

Reply:
It would make more sense it it was a modular structure, but this can be checked afterwards 👍